solutions/18.js

```diff
@@ -1,21 +1,27 @@
-// Find missing number in array
-// Return the missing number in an input array of consecutively
-// increasing numbers.
-// input: [5,6,8,9]
-// output: 7
+// Rahul Kalra
+// Return the  missing number in an input array of consecutively increasing number
+
+const solution3 = (array) =>{
+  let missNum = 0;
+  for( let i = 0; i < array.length; i++){
+    if( array[i+1]){
```

Comment:
you can remove this condition and include it as part of the for loop instead. `i+1 < array.length`